node/src/command.rs

```diff
@@ -468,7 +468,7 @@ pub fn run() -> Result<()> {
 						.map_err(Into::into)
 					},
 
-					ChainIdentity::Development => crate::service::start_parachain_node::<development_runtime::RuntimeApi, DevelopmentRuntimeExecutor>(
+					ChainIdentity::Development => crate::service::start_parachain_node_development::<development_runtime::RuntimeApi, DevelopmentRuntimeExecutor>(
```

Comment:
Maybe we should still re-use the `start_parachain_node_pendulum()` for this but pass the `development_runtime::RuntimeApi` to it?

Reply:
It will not work because development runtime does not implement `Zenlink Runtime Api`.
it means that for now it is not possible to pass development runtime to any function for other runtimes because of the trait bound
`C::Api: ZenlinkProtocolRuntimeApi<Block, AccountId, AssetId>,`